pkgs/development/tools/glpaper/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchhg, meson, ninja, pkg-config, wlroots, wayland, wayland-protocols
+, libX11, libGL }:
+
+stdenv.mkDerivation {
+  name = "glpaper";
+  version = "unstable-2020-03-30";
+
+  src = fetchhg {
+    url = "https://hg.sr.ht/~scoopta/glpaper";
+    rev = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
+    sha256 = "04y12910wvhy4aqx2sa63dy9l6nbs7b77yqpdhc96x2b3mgzgjfs";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config ];
+  buildInputs = [ wayland wayland-protocols wlroots libX11 libGL ];
```

Could you add a comment to the dependencies not listed in the `meson.build` file why they are needed?

---

By removing non-meson deps one by one this is the list. 
Compilation failed otherwise.

How should i comment this then?

---

wlroots or wayland-protocols do not seem to be required

```suggestion
  buildInputs = [
    wayland
    libX11 # required by libglvnd
    libGL
  ];
```